Content.Server/Administration/Systems/AdminSystem.cs

```diff
@@ -337,10 +337,15 @@ private void OnBabyJailMaxOverallMinutesChanged(int minutes)
 
     private void UpdatePanicBunker()
     {
-        var admins = PanicBunker.CountDeadminnedAdmins
-            ? _adminManager.AllAdmins
-            : _adminManager.ActiveAdmins;
-        var hasAdmins = admins.Any();
```

Comment:
why not use Any(session => ...)

also this should probably use ban flag instead

Reply:
The headmin said to use the Admin flag, not the Ban flag